rust/connlib/clients/apple/build.rs

```diff
@@ -131,6 +131,11 @@ fn link_swift(env: &Env) {
 }
 
 fn main() -> anyhow::Result<()> {
+    // Early exit build script to avoid errors on non-Apple platforms.
+    if !cfg!(target_vendor = "apple") {
```

Comment:
```suggestion
    if std::env::var("CARGO_CFG_TARGET_VENDOR").as_deref() != Ok("apple") {
```
`cfg!` has the same caveat as `cfg`, i.e. if you cross-compiled macOS -> Linux then `build.rs` would erroneously run.